### ISSUE TITLE
Sneak a peek on other users votes

### ISSUE THREAD

**Issue by vidare** [cthit/VoteIT#41]
As the UI is designed today it is (in my opinion) to easy to sneak a peak on other users (in front of you) screens during voting. Is there any plans of using forbidden color combinations (blue, black) to make it harder to see, or any similar solutions. Hopefully this is something we don't need, everyone is hopefully honest at IT.

**Comment by lindskogen** [comment]
No major research behind my comment here, but color-blind people could have problems using the interface if the contrast is too low?

**Comment by vidare** [comment]
Yes, that is true. A somewhat colorblind myself i would also opt for the UI as it is presented today (maybe somewhat smaller as default). But giving the possibility to "hide" your vote on the screen is something worth considering.

**Comment by abjugard** [comment]
A "colourblind-mode" could be implemented I guess, but generally the better solution is for users to just vote on their phones whenever possible.

**Comment by vidare** [comment]
Everyone already has their computer readily acessible, and therefore would use them primarily. My proposal is to "protect" the common user who don't care about these things.

**Comment by abjugard** [comment]
I can't speak for everyone, but being aware of the issue myself I instinctively used my phone anyway.

**Comment by vidare** [comment]
I myself (sitting in the far back during the meeting) can tell few peoples did use their phone. That sounds like a thing only an user aware of the problem would do. Or as I myself did during the vote: I didn't care (what if someone saw whom I voted for?) nobody else does (i think).

Therefore my proposal stands to protect the common user from this eventual "problem".

**Comment by lindskogen** [comment]
@vidare could you make a quick sketch, just so I could more easily visualize what you have in mind? :)

**Comment by Edholm** [comment]
We said quite clearly that people should vote with their phones (making it easier to hide whom you voted for). If people ignored that, then I assume that they didn't care if people saw who they voted for, thus it isn't really a problem.

**Comment by krabban91** [comment]
One way to make the interface look hideous, but still may confuse the sneak peaker, is to randomly select different colors for the buttons that can be selected on top of that the order of selections already is random. The mind subconsciously connects the name and color on their screen, hence will think that you (the person being watched) voted something completely different.
